index.js

```diff
@@ -91,57 +130,32 @@ async function runScript() {
     data,
   };
 
-  console.log(
-    `PREPARED REQUEST:`,
-    JSON.stringify(omit(request, "headers.Authorization"))
-  );
+  console.log(`PREPARED REQUEST:`, JSON.stringify(omit(request, 'headers.Authorization')));
   console.log(`SENDING...`);
 
   axios
     .request(request)
     .then((response) => {
       console.log(`REQUEST COMPLETED SUCCESSFULLY:`);
-      console.log("  ", response.status, response.statusText);
-      console.log("  ", "Headers:", JSON.stringify(response.headers));
-      console.log("  ", "Body:", JSON.stringify(response.data));
+      console.log('  ', response.status, response.statusText);
+      console.log('  ', 'Headers:', JSON.stringify(response.headers));
+      console.log('  ', 'Body:', JSON.stringify(response.data));
     })
     .catch((error) => {
       if (error.response) {
         // Response received with a non 20x status.
         console.log(`REQUEST RETURNED ERROR:`);
-        console.log("  ", error.response.status, error.response.statusText);
-        console.log("  ", "Headers:", JSON.stringify(error.response.headers));
-        console.log("  ", "Body:", JSON.stringify(error.response.data));
+        console.log('  ', error.response.status, error.response.statusText);
+        console.log('  ', 'Headers:', JSON.stringify(error.response.headers));
+        console.log('  ', 'Body:', JSON.stringify(error.response.data));
       } else {
         console.log(`ERROR SENDING REQUEST:`, error.message);
+
+        process.exitCode = 1;
```

Comment:
This exit code allows the pod to finish gracefully in a failure status, without the need to forcefully crash the process.